addons/survey/views/survey_user_views.xml

```diff
@@ -13,9 +13,10 @@
                 <filter name="completed" string="Completed" domain="[('state', '=', 'done')]"/>
                 <filter string="Partially Completed" name="partially_completed" domain="[('state', '=', 'skip')]"/>
                 <filter string="New" name="new" domain="[('state', '=', 'new')]"/>
+                <separator groups="base.group_no_one"/>
+                <filter string="Invite" name="invite" domain="[('input_type', '=', 'link')]" groups="base.group_no_one"/>
```

Comment:
Well I don't like filters in debug mode. Either we keep them either we remove them if they have no use. It is used to see people you invited on a survey form view, different from people that subscribed. Maybe it is not interesting, in which case it should be cleaned in master.

Reply:
I removed the "group_no_one" on it.
We will maybe question its use for the next saas but let's keep it for now.